src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -264,6 +263,18 @@ private static void PostConfigureBuilderOptions(
                 foreach (var (key, value) in envVars)
                 {
                     SetDefault(key, value);
+
+                    // See https://github.com/dotnet/runtime/blob/8edaf7460777e791b6279b395a68a77533db2d20/src/libraries/Microsoft.Extensions.Hosting/src/HostApplicationBuilder.cs#L96
```

Comment:
Why aren't we just calling `configuration.AddEnvironmentVariables(prefix: "DOTNET_");` and `configuration.AddEnvironmentVariables(prefix: "ASPNETCORE_");` ?

Reply:
Oh, because these env vars come from the launch profile, and aren't actually set in the environment variables.